src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,36 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int ACCEPT_THIS_AGE = 18;
+    private static final int MINIMUM_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        String login = user.getLogin();
+        String password = user.getPassword();
+        int age = user.getAge();
+
+        if (login == null) {
+            throw new RegisterException("This isn't login");
+        }
+        if (password == null) {
+            throw new RegisterException("This isn't password");
+        }
+        if (age < ACCEPT_THIS_AGE) {
+            throw new RegisterException("Ur age is too low");
```

Comment:
Please go through all your exception messages, remove slang words and make them in `Sentence case` - it is a small change, but keeps your code more readable and professional.